File: lua_lesson_1/ex02.lua


print(type("дед"))
print("тебе")
print("Мокар")
print("Улыбок")


local a = "дед"
local b = "Мокар"
local c = "Улыбок"
local d = "тебе"


print(a .. b .. c .. d)

print(a)
print(b)
print(c)
print(d)

print(a ..'\n'.. b ..'\n' .. c ..'\n' .. d)

print("Дед " .. "Мокар")
print("Дед Мокар")


print(a .. " Мокар")
print(a .. " ".. b)

local z = a .. " ".. b
print(z)

print(z .. ", " .. c .." ".. d)

local y = z .. ", " .. c .." ".. d
print(y .. "!")

y = y .. "!"
print(y)

File: Python_lessons/python_lesson_4/ex02.py
# Переписать программу с использования goto на использование цикла
# с циклом while
# c циклом repeat ... until
# c циклом for

# local i = 0
# ::loop::
# if i > 100 then
#     goto finish
# end
# print(i)
# i = i + 1
# goto loop
# ::finish::

# for
for i in range(101):
    print(i)
    
    
    

i = 0
while True:
    print(i)
    i += 1
    if i > 100:
        break
    
    
    
# while
i = 0
while i <= 100:
    print(i)
    i += 1